Processor_CashFlows_SftpCommunication.php

```diff
@@ -0,0 +1,64 @@
+<?php
+class Processor_CashFlows_SftpCommunication extends Jobs_SftpCommunication
+{
+    public function __construct($JobType, $FilenamePattern, $LogJob, $Verbose)
+    {
+        parent::__construct(
+            $JobType,
+            $FilenamePattern,
+            $LogJob,
+            Jobs_Chargeback_Cashflows::CASHFLOWS_SFTP_FILE_LIVESPAN,
+            $Verbose
+        );
+        $this->ProcessorName = 'CashFlows';
+        $this->setDoneDir(Jobs_Chargeback_Cashflows::DIR_TYPE_DONE);
+        $this->setCacheDir(Jobs_Chargeback_Cashflows::DIR_TYPE_CACHE);
+    }
+
+    public function setCommand($cmdDirectory)
+    {
+        $command = $cmdDirectory . DIRECTORY_SEPARATOR . "cashflows.sftp.exp " .
+            Jobs_Chargeback_Cashflows::CASHFLOWS_SFTP_SERVER . ' ' .
+            Jobs_Chargeback_Cashflows::CASHFLOWS_SFTP_PORT . ' ' .
+            Jobs_Chargeback_Cashflows::CASHFLOWS_SFTP_SERVER_USERNAME . ' ' .
+            Jobs_Chargeback_Cashflows::CASHFLOWS_SFTP_SERVER_PASSWORD;
+            if ($this->Verbose) echo "SFTP command: " . $command . " \n ";
+        parent::setCommand($command);
+    }
+    protected function getJobDirectory($JobType, $DirectoryType)
```

Comment:
This method seems like a good opportunity to consider a solution involving polymorphism.

If we extend `Jobs_Chargeback_Cashflow` then we can create new subtypes each for `CBK`, `FRAUD` and `RECONCILE`.

The repetitive code inside of each of the `case` statements can be replaced with a single call to a common method shared by each of these classes to get the string.

Something like this:
```php
function getJobDirectory(Jobs_Chargeback_Cashflow $Job, $DirectoryType)
{
    return $Job->getJobDirectory($DirectoryType);
}
```

Even better, consider just calling `getJobDirectory()` on the objects directly in-place, if possible.